Doc/Tutorial/chapter_learning.tex

```diff
@@ -78,42 +78,26 @@ \subsection{Training the logistic regression model}
 %doctest . lib:numpy
 \begin{minted}{pycon}
 >>> from Bio import LogisticRegression
->>> xs = [[-53, -200.78],
-...       [117, -267.14],
-...       [57, -163.47],
-...       [16, -190.30],
-...       [11, -220.94],
-...       [85, -193.94],
-...       [16, -182.71],
-...       [15, -180.41],
-...       [-26, -181.73],
-...       [58, -259.87],
-...       [126, -414.53],
-...       [191, -249.57],
-...       [113, -265.28],
-...       [145, -312.99],
-...       [154, -213.83],
-...       [147, -380.85],
-...       [93, -291.13]]
-...
->>> ys = [1,
-...       1,
-...       1,
-...       1,
-...       1,
-...       1,
-...       1,
-...       1,
-...       1,
-...       1,
-...       0,
-...       0,
-...       0,
-...       0,
-...       0,
-...       0,
-...       0]
-...
+>>> xs = [
+...     [-53, -200.78],
+...     [117, -267.14],
+...     [57, -163.47],
+...     [16, -190.30],
+...     [11, -220.94],
+...     [85, -193.94],
+...     [16, -182.71],
+...     [15, -180.41],
+...     [-26, -181.73],
+...     [58, -259.87],
+...     [126, -414.53],
+...     [191, -249.57],
+...     [113, -265.28],
+...     [145, -312.99],
+...     [154, -213.83],
+...     [147, -380.85],
+...     [93, -291.13],
+... ]
+>>> ys = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
We can keep this as one value per line if preferred by adding a trailing comma (black convention).